types/ramda/index.d.ts

```diff
@@ -1605,6 +1605,8 @@ export function pickAll(names: readonly string[]): <T, U>(obj: T) => U;
 /**
  * Returns a partial copy of an object containing only the keys that satisfy the supplied predicate.
  */
+export function pickBy(pred: ObjPred, record: Record<keyof any, any>): Record<keyof any, any>;
+export function pickBy(pred: ObjPred): (record: Record<keyof any, any>) => Record<keyof any, any>;
```

Comment:
Why are you using any instead of unknown? Any is less safe.

Reply:
Yes, `unknown` is better, I will fix that.

Reply:
No, I am sure that `any` is better(If you try to it with `filter` in `pipe`).
```ts
export interface Dictionary<A> {
    [index: string]: A;
}
```

Do you feel that this project tried to describe the type accurately so hard, but in the end it made it terrible?

Anytime, at any time, we have written a *correct* code, but what we get is a type error, and then we explicitly assert the type.

Why do we do this on earth only to make our job harder?

This project is so crazy that using Typescript  to describe Clojure like Haskell.

No! We don't need the *perfect* type, just make things work!